proto/dream11/od/service/v1/service.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+package dream11.od.service.v1;
+
+import "dream11/od/dto/v1/service.proto";
+
+option go_package = "github.com/dream11/odin/proto/gen/go/dream11/od/service/v1";
+
+service ServiceService {
+  rpc DescribeService(DescribeServiceRequest) returns (DescribeServiceResponse) {}
+}
+
+message DescribeServiceRequest {
+  string service_name = 1;
```

Comment:
Should we be consistent in naming these eg. either have `name` and `version` OR `service_name` and `service_version`?